components/KiboConfirmOrder.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <div class="sf-confirm-order" data-testid="confirm-order">
+    <SfHeading
+      :title="$t('Order Details')"
+      :level="2"
+      class="sf-heading--left sf-heading--no-underline title"
+    />
+    <KiboOrderLineItems :order="order" />
+
+    <hr class="hr-order" />
+    <div class="order-total">
+      <SfProperty
+        :name="$t('Subtotal')"
+        :value="$n(order.subtotal, 'currency')"
```

Comment:
use KiboPrice using slots

Reply:
Will integrate later once it becomes available in develop branch from your PR.